pmm/pmm2-api-tests.groovy

```diff
@@ -73,72 +69,88 @@ pipeline {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Start staging') {
+        stage('Setup') {
             steps {
+                sh '''
+                    sudo curl -L https://github.com/docker/compose/releases/download/1.29.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/bin/docker-compose > /dev/null
+                    sudo chmod +x /usr/bin/docker-compose
+                    sudo usermod -aG docker ec2-user
```

Comment:
This line is not needed, it's already set on worker: https://github.com/Percona-Lab/jenkins-pipelines/blob/master/IaC/pmm.cd/init.groovy.d/cloud.groovy#L290

Reply:
okay trying, will remove if the job is green.

Reply:
I have tested and it works.

Reply:
thank you